allauth/socialaccount/providers/eventbrite/provider.py

```diff
@@ -39,5 +47,16 @@ def extract_common_fields(self, data):
             image_url=data.get('image_url')
         )
 
+    def extract_email_addresses(self, data):
+        """Extract email addresses if verified."""
+        ret = []
+        emails = data.get('emails')
+        for email_account in emails:
+            if email_account['verified']:
```

Comment:
Ah ok, sorry this was following another example (google's I think). Much prefer to keep all. Will fix.